### ISSUE TITLE
python2 incompatibility in `setup.py` (visdom==0.1.8.7)

### ISSUE THREAD

**Issue by menshikh-iv** [facebookresearch/visdom#548]
**Bug Description**
Broken compatibility with `python2`.
The issue with `readme` variale in `setup.py`

With `0.8.1.6`: `readme = open('README.md').read()`
With `0.8.1.7`: `readme = open('README.md', 'rt', encoding='utf8').read()`

`encoding` argument not supported by `python2` (+ I guess `t` also not supported by `python2`)

**Reproduction Steps**
Enter steps to reproduce the behavior:
1. Create virtualenv with `python2.7`
2. `pip install visdom==0.1.8.7`

**Expected behavior**
Package installed succesfully

**Acrtual behavior**
```
ivan@P50:~$ pip install visdom==0.1.8.7
Collecting visdom
  Downloading https://files.pythonhosted.org/packages/85/09/be60dfe97d9ed8c8b17c40fa34cbe50b4652e346b697223846ab0030b9ae/visdom-0.1.8.7.tar.gz (1.4MB)
    100% |████████████████████████████████| 1.4MB 1.2MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-jqLhlI/visdom/setup.py", line 35, in <module>
        readme = open('README.md', 'rt', encoding='utf8').read()
    TypeError: 'encoding' is an invalid keyword argument for this function
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-jqLhlI/visdom/
```

**Comment by ap--** [comment]
Our automated python environment tests hit the same bug today...

**Comment by pietroastolfi** [comment]
Same bug.
I temporarly solved by adding to setup.py the following line:
`from io import open`

**Comment by JackUrb** [comment]
Fixed in 1.8.8, which has been uploaded to PyPI
